### PR TITLE
Scapri/sandrarevert

### PR DESCRIPTION
**Pull request by scapri**

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 main()
 {
-   printf("version 1\n");
-   printf("changed code\n");
+   printf("version 10 - release code\n");
+   printf("for the pr demo\n");
```
Comment (ciaodog) [inline main.c]: Shouldn't this be capitalized?

File: isyourname.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+how great
```
Comment (ciaodog) [inline isyourname.txt:1]: Yes he is

**Review by ciaodog — CHANGES_REQUESTED** [review]
This feature looks good, but please make the changes in my review.

File: isyourname.txt (hunk 2 above)
Comment (scapri) [reply isyourname.txt:1]: 👍

File: main.c (hunk 1 above)
Comment (scapri) [reply main.c]: Oops, yes, I'll fix that

**Review by ciaodog — APPROVED** [review]
Good change, thanks

**Pull request merged**